schemas/2018-08-20-preview/Microsoft.HealthcareApis.json

```diff
@@ -171,115 +137,100 @@
-                  }
-                ]
+                "type": "string"
               }
             },
             {
               "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
             }
-          ]
+          ],
+          "description": "The origins to be allowed via CORS."
         },
         "headers": {
-          "readOnly": false,
           "oneOf": [
             {
               "type": "array",
-              "description": "The headers to be allowed via CORS.",
               "items": {
-                "anyOf": [
-                  {
-                    "type": "string",
-                    "description": "A header for CORS."
-                  },
-                  {
-                    "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
-                  }
-                ]
+                "type": "string"
               }
             },
             {
               "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
             }
-          ]
+          ],
+          "description": "The headers to be allowed via CORS."
         },
         "methods": {
-          "readOnly": false,
           "oneOf": [
             {
               "type": "array",
-              "description": "The methods to be allowed via CORS.",
               "items": {
-                "anyOf": [
-                  {
-                    "type": "string",
-                    "description": "A method for CORS.",
-                    "enum": [
-                      "DELETE", "GET", "OPTIONS", "PATCH", "POST", "PUT"
-                    ]
-                  },
-                  {
-                    "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
-                  }
-                ]
+                "type": "string"
               }
             },
             {
               "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
             }
-          ]
+          ],
+          "description": "The methods to be allowed via CORS."
         },
         "maxAge": {
-          "readOnly": false,
           "oneOf": [
             {
               "type": "integer",
-              "description": "The max age to be allowed via CORS.",
               "minimum": 0,
               "maximum": 99999
             },
             {
               "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
             }
-          ]
+          ],
+          "description": "The max age to be allowed via CORS."
         },
         "allowCredentials": {
-          "readOnly": false,
```

Comment:
We need readonly field in all the properties. Please restore them back.

Reply:
See above comment on readOnly